app/models/saved_claim/veteran_readiness_employment_claim.rb

```diff
@@ -161,20 +171,28 @@ def send_to_central_mail!(user)
 
     update!(form: form_copy.to_json)
     log_message_to_sentry(guid, :warn, { attachment_id: guid }, { team: 'vfs-ebenefits' })
-    @sent_to_cmp = true
-    log_to_statsd('cmp') do
-      process_attachments!
-    end
 
     send_central_mail_confirmation_email(user)
+    @sent_to_cmp = true
+  rescue => e
+    log_message_to_sentry('Error uploading VRE claim to central mail',
```

Comment:
I believe we're still moving away from sentry logging to just use the Rails.logger as it's logs are tied to Datadog